lib/momentum/opsworks.rb

```diff
@@ -77,6 +77,27 @@ def initialize(aws_id, aws_secret)
       @ow = Momentum::OpsWorks.client(aws_id, aws_secret)
     end
 
+    def execute_recipe!(stack_name, layer, recipe, app_name = Momentum.config[:app_base_name])
```

Comment:
The reason why I just added it here, is because OpsWorks lingo seems to consider this a 'deploy'. The call to the AWS API is `create_deployment` with the command being an 'execute_recipes', which also kinda mirrors the UI - ie, deploys and commands/recipes are all 'deploys' and viewed/triggered in the Deployments section.